docs/interfaces/autoupdate-service.txt

```diff
@@ -119,19 +119,22 @@ Interface RelationFieldDescriptor extends Fields {
  * null` without using this interface, the response would be just the template
  * field `B_$_ids` and no structured fields.
  *
- * For just resolving values, leav `values` out (hence the optional parameter).
+ * For just resolving values, leave `values` out (hence the optional parameter).
+ *
+ * For retrieveing a single structured field, use `RelationFieldDescriptor`. E.g.
+ * when trying to get the groups for a user from meeting 2, use `group_2_ids`
+ * directly and do not bother with structured fields. The terminology is a
+ * `specific structured field`.
  *
  * If the values of all structured fields are references, use the `values`
  * parameter analog to the `fields` paramter as documented in
  * `GenericRelationFieldDescriptor`.
  */
 Interface StructuredFieldDecriptor {
     type: 'template',
-    values?: {
-        [field: Field]: GenericRelationFieldDescriptor
-            | RelationFieldDescriptor
-            | StructuredFieldDescriptor
-            | null;
+    values?: GenericRelationFieldDescriptor
```

Comment:
I would suggest renaming it to value if it's only holding a single descriptor now

Reply:
The meaning is "How to handle possible values?"; this is currently fixed in the autoupdate serivce and client. I'm not sure if this is worth the renaming. @ostcar ?